LedRGB.cpp

```diff
@@ -9,21 +9,15 @@ LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
 }
 
 void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
-    r /= 2.55;
-    g /= 2.55;
-    b /= 2.55;
-    if (checkBrightness) {
-        RedLed.setBrightness(r);
-        GreenLed.setBrightness(g);
-        BlueLed.setBrightness(b);
-    }
+    setBrightness(r);
+    setBrightness(g);
+    setBrightness(b);
```

Comment:
The overall brightness is applied evenly across the individual brightnesses. 
Imagine we set the led to a light yellow, brightness starts at 100%:
`LedRGB myRGB;`
`myRGB.setRGB(255,255,50);`
What are the brightnesses of each led?

now we keep the same color and lower the brightness of the LedRGB.
`myRGB.setBrightness(50);`

now the color must stay the same but be 50% as bright.